### PR TITLE
add isBSALoaded

### PR DESCRIPTION
**Pull request by c6-dev**

### CODE REVIEW

**Comment by Senjay-id** [comment]
:pepesmile:

**Review by lStewieAl — APPROVED** [review]

File: JG/functions/fn_file.h [hunk 1]
```diff
@@ -10,8 +10,23 @@ DEFINE_COMMAND_PLUGIN(GetTextureFormat, , 0, 1, kParams_OneString);
 DEFINE_COMMAND_PLUGIN(GetTextureMipMapCount, , 0, 1, kParams_OneString);
 DEFINE_COMMAND_PLUGIN(PlaySoundFile, , 0, 3, kParams_OneString_TwoOptionalInts);
 DEFINE_COMMAND_PLUGIN(StopSoundFile, , 0, 0, NULL);
+DEFINE_COMMAND_PLUGIN(IsBSALoaded, , 0, 1, kParams_OneString);
 #include <filesystem>
 
+bool Cmd_IsBSALoaded_Execute(COMMAND_ARGS) {
+	char path[MAX_PATH];
+	char fixPath[260];
```
Comment (lStewieAl) [inline JG/functions/fn_file.h:18]: should also use MAX_PATH

File: JG/functions/fn_file.h [hunk 2]
```diff
@@ -10,8 +10,23 @@ DEFINE_COMMAND_PLUGIN(GetTextureFormat, , 0, 1, kParams_OneString);
...
+	*result = 0;
+	if (ExtractArgsEx(EXTRACT_ARGS_EX, &path)) {
+		sprintf(fixPath, "DATA\\%s", path);
```
[the remark this answers is not in the document]
Comment (c6-dev) [reply JG/functions/fn_file.h]: fixed

File: JG/functions/fn_file.h (hunk 1 above)
Comment (c6-dev) [reply JG/functions/fn_file.h]: fixed

**Pull request merged**